pom.xml

```diff
@@ -34,7 +34,7 @@
           ${project.basedir}/src/main/resources/swagger.api/bursar-export.yaml
         </bursar-export.yaml.file>
 
-        <folio-spring-base.version>1.0.5</folio-spring-base.version>
+        <folio-spring-base.version>2.0.0</folio-spring-base.version>
         <folio-spring-tenant.version>1.0.0</folio-spring-tenant.version>
         <folio-data-export-spring.version>1.1.2</folio-data-export-spring.version>
```

Comment:
mod-data-export-spring should be migrated to spring-base 2.0 before. Also I want to point out spring-exp-module has CQL repository/services as we are going to use  spring-base 2.0 - we should use CQL support from this library